yarn.lock

```diff
@@ -2334,46 +2334,46 @@
     write-file-atomic "^3.0.3"
 
 "@loaders.gl/3d-tiles@^3.0.8":
```

Comment:
The yarn.lock has changed a lot. We need to check what happened here

Reply:
https://github.com/CartoDB/carto-react/pull/253